src/generate.c

```diff
@@ -195,11 +197,13 @@ find_interface(gchar* interface, GHashTable* netdefs)
     return ret;
 }
 
-#define CHECK_CALL(call) {\
-    if (!call) {\
+#define CHECK_CALL(call, ignore_errors) {\
+    if (!call && !ignore_errors) {\
         error_code = 1; \
         fprintf(stderr, "%s\n", error->message); \
         goto cleanup;\
+    } else if (error && ignore_errors) {\
+        g_clear_error(&error); \
```

Comment:
suggestion: We should consider still logging the error here, as least to debug log.. Maybe using an `IGNORED: ...` prefix?